### PR TITLE
changes in url genration algorithm for iterative server probing.

### PR DESCRIPTION
**Pull request by hq9000**
https://github.com/NodeGuy/ServerDate/issues/32

### CODE REVIEW

File: lib/ServerDate.js [hunk 1]
```diff
@@ -1,24 +1,17 @@
 /*
-
```
Comment (NodeGuy) [inline lib/ServerDate.js]: Please don't remove these whitespace lines.

File: lib/ServerDate.js [hunk 2]
```diff
@@ -122,7 +115,7 @@ Object.defineProperty(ServerDate, "synchronizationIntervalDelay", {
 
 // After the initial synchronization the two clocks may drift so we
 // automatically synchronize again every synchronizationIntervalDelay.
-ServerDate.synchronizationIntervalDelay = 60 * 60 * 1000; // ms
+ServerDate.synchronizationIntervalDelay = 10 * 60 * 1000; // ms, 10 minutes
```
Comment (NodeGuy) [inline lib/ServerDate.js]: You're worried about clock drift happening within 10 minutes?

Comment (hq9000) [inline lib/ServerDate.js]: Not that much, in most cases, in fact. But considering how easy is to sync (anyway, we do it 10 times after initial load) and the fact there are environments where the drift could be significant (imagine running browser within virtual machine having poor syncronisation with the host) I think we can afford doing sync more often. What do you think?

Comment (NodeGuy) [inline lib/ServerDate.js]: OK, that's fine with me.

**Comment by hq9000** [comment]
Ok, is anything left that I have to do about this change?

**Comment by hq9000** [comment]
Okay, it seems the blanks are fixed now.

**Comment by NodeGuy** [comment]
Thank you.

**Pull request merged**